src/Components/Aspire.Elastic.Clients.Elasticsearch/AssemblyInfo.cs

```diff
@@ -0,0 +1,7 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Aspire;
+using Aspire.Elastic.Clients.Elasticsearch;
+
+[assembly: ConfigurationSchema("Aspire:Elastic:Clients:Elasticsearch", typeof(ElasticClientsElasticsearchSettings))]
```

Comment:
Note that I removed the LoggingCategory because I didn't see anything in the library that would log to the ILogger. Am I missing something? Or is there no logging support for ElasticsearchClient.